doc/source/rllib/rllib-concepts.rst

```diff
@@ -230,76 +230,60 @@ First, check out the `PPO trainer definition <https://github.com/ray-project/ray
 
         @staticmethod
         @override(Trainer)
-        def execution_plan(workers, config, **kwargs):
+        def training_iteration(workers, config, **kwargs):
             ...
 
-Besides some boilerplate for defining the PPO configuration and some warnings, the most important method to take note of is the ``execution_plan``.
+Besides some boilerplate for defining the PPO configuration and some warnings, the most important method to take note of is the ``training_iteration``.
 
-The trainer's `execution plan <#execution-plans>`__ defines the distributed training workflow.
+The trainer's `training iteration function <#training-iteration>`__ defines the distributed training workflow.
```

Comment:
This does not work I think.

Reply:
I mean: There is no training-iteration section here to which it leads me.

Reply:
fixed